RedHatInsights/malware-detection-frontend
# fix(RHINENG-10450): fix redux intit, clean up fec.config

Resolves: https://issues.redhat.com/browse/RHINENG-10450

This is to make the inventory table stable by properly initializing Redux. There were also a few leftovers from FEC-config migration, this PR cleans up them as well.

To test:
1. Play with the systems page
2. Make sure you do not see issues related to inventory fed-module, except those asset loading ones
